content/concept-amendments.md

```diff
@@ -86,13 +86,139 @@ When an amendment gets enabled for the network after the voting process, servers
 * Do not participate in the consensus process
 * Do not vote on future amendments
 
-Becoming amendment blocked is a security feature to protect backend applications. Rather than guessing and maybe misinterpreting a ledger after new rules have applied, `rippled` reports that it does not know the state of the ledger because it does not know how the amendment works.
+Becoming amendment blocked is a security feature to protect backend applications. Rather than guessing and maybe misinterpreting a ledger after new rules have been applied, `rippled` reports that it does not know the state of the ledger because it does not know how the amendment works.
```

Comment:
After re-reading this in context, I think I would suggest changing "backend applications" to "applications that depend on XRP Ledger data". My rationale is that "backend" is a fairly vaguely defined term which people tend to use in varying ways, and also because JavaScript web-apps (typically considered "frontend" code) can connect to `rippled` directly via WebSocket, but amendment blocking is meant to protect them, too.